authBackEnd/apps/common/mixins.py

```diff
@@ -1,11 +1,10 @@
 from django.shortcuts import redirect, reverse
 
 
-class LoginRequiredMixin(object):  # these are kinda mysterious - how the fuck do they work / what do they do?
```

Comment:
The login MIxin is basically to make sure that the user is signed in before viewing the dashboard. It won't let the user to see the dashboard without being logged in

Reply:
Makes sense!